docs/reference/sql/limitations.asciidoc

```diff
@@ -118,6 +118,17 @@ SELECT * FROM test GROUP BY age ORDER BY COUNT(*) LIMIT 100;
 It is possible to run the same queries without a `LIMIT` however in that case if the maximum size (*10000*) is passed,
 an exception will be returned as {es-sql} is unable to track (and sort) all the results returned.
 
+Moreover, the aggregation(s) used in the `ORDER BY`, must be only plain aggregate functions. No scalar
```

Comment:
"_must be only_ plain aggregate functions" could be rephrased to "can only be ..."